test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamples.cs

```diff
@@ -395,7 +395,7 @@ public void TrainOnAutoGeneratedData()
             // Turn the data into the ML.NET data view.
             // We can use CreateDataView or CreateStreamingDataView, depending on whether 'churnData' is an IList, 
             // or merely an IEnumerable.
-            var trainData = mlContext.CreateStreamingDataView(churnData);
+            var trainData = mlContext.Data.ReadFromEnumerable(churnData);
```

Comment:
>ReadFromEnumerable [](start = 43, length = 18)

please update cookbook.md file as well.

Reply:
Oops. Sorry missed this one.

---
In reply to: [249606989](https://github.com/dotnet/machinelearning/pull/2189#discussion_r249606989) [](ancestors = 249606989)